app/websocket/app/main/model_manager.py

```diff
@@ -181,9 +184,11 @@ def list_models(detailed_list: bool=False):
         if detailed_list is True:
             models = [model.__dict__ for model in result]
             for model in models:
-                model['model_size'] = "{}KB".format(sys.getsizeof(model['model'])/1000)
-                model.pop('model')
-                model.pop('_sa_instance_state')
+                model["model_size"] = "{}KB".format(
+                    sys.getsizeof(model["model"]) / 1000
+                )
+                model.pop("model")
```

Comment:
why is this .pop needed? Can you add some comments on what's going on here?

Reply:
because we don't want to return the model in json. So replacing the model info with model_size.